main.go

```diff
@@ -103,15 +103,16 @@ var (
 	s3ScanBuckets  = s3Scan.Flag("bucket", "Name of S3 bucket to scan. You can repeat this flag.").Strings()
 
 	gcsScan           = cli.Command("gcs", "Find credentials in GCS buckets.")
-	gcsProjectID      = gcsScan.Flag("project-id", "GCS project ID used to authenticate. Can be provided with environment variable GOOGLE_CLOUD_PROJECT.").Envar("GOOGLE_CLOUD_PROJECT").String()
+	gcsProjectID      = gcsScan.Flag("project-id", "GCS project ID used to authenticate. Can NOT be used w/ unauth scan. Can be provided with environment variable GOOGLE_CLOUD_PROJECT.").Envar("GOOGLE_CLOUD_PROJECT").String()
 	gcsCloudEnv       = gcsScan.Flag("cloud-environment", "Use Application Default Credentials, IAM credentials to authenticate.").Bool()
 	gcsServiceAccount = gcsScan.Flag("service-account", "Path to GCS service account JSON file.").ExistingFile()
 	gcsWithoutAuth    = gcsScan.Flag("without-auth", "Scan GCS buckets without authentication. This will only work for public buckets").Bool()
 	gcsAPIKey         = gcsScan.Flag("api-key", "GCS API key used to authenticate. Can be provided with environment variable GOOGLE_API_KEY.").Envar("GOOGLE_API_KEY").String()
-	gcsIncludeBuckets = gcsScan.Flag("include-buckets", "Buckets to scan. You can repeat this flag. Globs are supported").Short('i').Strings()
-	gcsExcludeBuckets = gcsScan.Flag("exclude-buckets", "Buckets to exclude from scan. You can repeat this flag. Globs are supported").Short('x').Strings()
-	gcsIncludeObjects = gcsScan.Flag("include-objects", "Objects to scan. You can repeat this flag. Globs are supported").Short('I').Strings()
-	gcsExcludeObjects = gcsScan.Flag("exclude-objects", "Objects to exclude from scan. You can repeat this flag. Globs are supported").Short('X').Strings()
+	gcsIncludeBuckets = gcsScan.Flag("include-buckets", "Buckets to scan. Comma seperated list of buckets. Globs are supported").String()
```

Comment:
I couldn't use it w/ .String(), and after talking to a couple people in the community as well as personal use the repeated flag seemed a lot more cumbersome than a command separated string. Most users (including myself) either wanted to pass in 1 bucket or a lot. It was rare to find someone only wanting to pass in a couple and even then having to repeat the flag seemed a bit tedious.

Happy to put both though.

Reply:
Oh that's weird that a short flag can't be used with `.String()`. I don't see a downside to allowing both multi-flag and comma separated. Maybe some people prefer multiple flags for some reason?

That said, I don't have a strong opinion either way, so if you want to keep just one flag that's cool too.

Reply:
hmm.... i'm actually not sure if Kingpin lets you have multiple value types. Were you able to do it? If so would you mind providing an example of how you got it to work.

Reply:
Oh, I thought it could be the old value

```go
gcsExcludeObjects = gcsScan.Flag("exclude-objects", "Objects to exclude from scan. You can repeat this flag. Globs are supported").Short('X').Strings()
```

Here's a quick example:

```go
var fooFlags = kingpin.Flag("foo", "Foo mode.").Short('f').Strings()

func main() {
	kingpin.Parse()
	var foos []string
	for _, flag := range *fooFlags {
		foos = append(foos, strings.Split(flag, ",")...)
	}
	fmt.Printf("%q\n", foos)
}
```

```
» go run main.go --foo 'bar, baz' -f test
["bar" " baz" "test"]
```

Reply:
oh that's what you meant. Got it.